packages/tabs/src/Tabs.spec.tsx

```diff
@@ -252,12 +295,12 @@ describe('packages/tabs', () => {
           </Tabs>
         </>,
       );
+      testUtils = getTestUtils();
     });
 
     test('only the current Tabs set is toggled when the arrow keys are pressed', () => {
-      const { getAllByTestId } = renderResult;
-      const [tab1A] = getAllByTestId('tab-1-a');
-      const [tab1B] = getAllByTestId('tab-1-b');
+      const { getAllTabsInTabList } = testUtils;
```

Comment:
If I'm reading this correctly, this is testing only the first `Tabs`. Is that correct?

Reply:
wrote this out more explicitly